app/Http/Controllers/Admin/CaveCrudController.php

```diff
@@ -126,11 +126,16 @@ public static function getMonsterSubfields()
             MonsterCrudController::getFieldsArrayForMiscellaneousTab(),
         );
 
+        $unsuportedRelations = ['dummyproducts', 'recommends', 'stars', 'bills', 'ball', 'wish', 'postalboxes'];
```

Comment:
Hmm... why are these unsupported? I would rather we filter by what is not supported in those fields, rather than the names themselves... eg. by `type`, by `relationType`, by `subfields` or something like that. That would better tell future-us (and anyone looking at the demo code) what in particular is not supported.